src/otx/algorithms/detection/adapters/openvino/task.py

```diff
@@ -468,8 +449,15 @@ def load_inferencer(
             async_requests_num,
         ]
         if self.task_type == TaskType.DETECTION:
+            if (
+                self.task_environment.model_template.model_template_id == "Custom_Object_Detection_YOLOX"
+                and not self.config.tiling_parameters.enable_tiling
+            ):
+                args.append({"resize_type": "fit_to_window_letterbox", "pad_value": 114})
             inferencer: BaseInferencerWithConverter = OpenVINODetectionInferencer(*args)
         if self.task_type == TaskType.INSTANCE_SEGMENTATION:
+            if self.config.tiling_parameters.enable_tiling:
+                args.append({"resize_type": "fit_to_window_letterbox"})
```

Comment:
These setting might be decided from preprocessing options in the future :)

Reply:
Right now we have no an appropriate mechanism to forward such parameters from algo backend config to inference, but when we fully switch to ModelAPI, that will be moved to the task level. However, that's bot backward compatible. Or which mechanism do you mean?